### PR TITLE
adds faq section 

### LINKED ISSUE #192: Create FAQ doc based on existing FAQ doc
(This has already been worked on, but it didn't have an issue. All PRs should be based on existing issues.)

### PR DESCRIPTION
**Pull request by ArthurSampaio**
I moved faq section in the old docs to the new docs. 

closes #192

### CODE REVIEW

File: faq.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
+FAQ
+==============
+
+Truth be told, these questions aren't frequently asked. Hopefully the answers are still helpful.
```
Comment (adammichaelwood) [inline faq.rst]: Let's delete this. It's silly and not in keeping with the tone of our docs.

File: faq.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
+FAQ
+==============
+
+Truth be told, these questions aren't frequently asked. Hopefully the answers are still helpful. 
+
+.. _what-is-ODK:
```
Comment (adammichaelwood) [inline faq.rst]: Don't capitalize section labels.

File: faq.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+What is ODK? 
+~~~~~~~~~~~~~~~~~~
+
+Open Data Kit (ODK) is a free and open-source set of tools which help organizations create mobile data collection solutions.
```
Comment (adammichaelwood) [inline faq.rst]: no hyphen in "open source"

File: faq.rst [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+Our `blog <https://opendatakit.org/blog/>`_ blog and `deployments page <https://opendatakit.org/about/deployments/>`_ have good examples of what others have used ODK for. Our `research page <https://opendatakit.org/about/research/>`_ also has videos and slides that explain ODK.
+
+.. _how-use: 
+
+How do I use ODK?
+~~~~~~~~~~~~~~~~~~~~
+
+Please read through the documentation on our `implementer instructions <https://opendatakit.org/use/>`_ and on the `developer wiki <https://github.com/opendatakit/opendatakit/wiki>`_.
```
Comment (adammichaelwood) [inline faq.rst]: Should link to the home page of the docs.

File: faq.rst [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+If that doesn't help, search the `ODK forum <https://forum.opendatakit.org/>`_ to see if the issue has been answered.
+
+If you don't find an answer after searching, then post your question to the appropriate category.
+
+.. _what-kinds-question:
+
+What kinds of questions do you answer on the mainling list?
```
Comment (adammichaelwood) [inline faq.rst]: mailing

File: faq.rst [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+If that doesn't help, search the `ODK forum <https://forum.opendatakit.org/>`_ to see if the issue has been answered.
+
+If you don't find an answer after searching, then post your question to the appropriate category.
+
+.. _what-kinds-question:
+
+What kinds of questions do you answer on the mainling list? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our team only answers questions about the `supported tools <https://opendatakit.org/about/tools/>`_.
```
Comment (adammichaelwood) [inline faq.rst]: If this is a single paragraph, it should be one line of source code.
If it is three paragraphs, they need blank lines between.

File: faq.rst [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+An ODK tool isn't working the way I expected. What do I do? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Make sure you are running the latest versions of ODK software. You can find them under our `downloads <https://opendatakit.org/downloads/>`_ page.
+
+Search the `issue tracker <https://github.com/opendatakit/opendatakit/issues>`_ to see if the problem you are having has been reported. If you can't find it, report it there.
+
+Please be precise about the tool you are using, the version of the software, and include all the steps you did so we can reproduce the same problem. 
+If you can get a stack trace, please attach it. If the problem is with a form, please attach it as well. If you are new to filing bug reports, read How to Report Bugs Effectively.
```
Comment (adammichaelwood) [inline faq.rst]: paragraph break? or not?

File: faq.rst [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+See `Collect Troubleshooting <https://github.com/opendatakit/opendatakit/wiki/Collect-Troubleshooting>`_. Capturing an error log is more difficult now that the newer Android operating systems restrict access to the system log stream. 
+Once you begin capturing a log and have reproduced the crash, please open an issue `here <https://github.com/opendatakit/opendatakit/issues>`_ and attach the captured log file.
+
+.. _request-new-feature: 
+
+How do I request a new feature or enhacement?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Search the `issue tracker <https://github.com/opendatakit/opendatakit/issues>`_ to see if the feature you want has been suggested. If you find it, vote for it by adding yourself to receive notifications. 
+If you don't find it, open a new issue (feature request) describing the scenario which you need the feature for.
```
Comment (adammichaelwood) [inline faq.rst]: paragraph break? or no?

File: faq.rst [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+
+.. _customize-ODK-solution: 
+
+How do I customize or implement an ODK solution? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The core team does not provide code or implementation support beyond what is described on our `implementer site <https://opendatakit.org/>`_ and on the developer site. 
+If you need more support, `hire help <https://opendatakit.org/help/help-for-hire/>`_ from one of the ODK implementations companies.
```
Comment (adammichaelwood) [inline faq.rst]: paragraph break? or no?

File: faq.rst [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+Other publications to cite for individual ODK tools are available on the `Research <https://opendatakit.org/about/research/>`_ page.
+
+.. _other-data-collection-to-consider: 
+
+Are there other data collection systems I should consider? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Of course! There are many other data collection systems that might work better for you.
+Some of them use ODK in some way (`Ona <http://ona.io/>`_, `Survey CTO <http://www.surveycto.com/>`_, `KoBo Toolbox <http://kobotoolbox.org/>`_, `Commcare HQ <http://commcarehq.com/>`_, `DoForms <http://doforms.com/>`_, `DataWinners <http://datawinners.com/>`_, `ViewWorld <http://viewworld.dk/>`_, `PhiCollect <http://webfirst.com/phicollect>`_) while others are ODK compatible (`JavaRosa <http://www.dimagi.com/javarosa/>`_, `OpenXData <http://www.openxdata.org/>`_, `RapidSMS <http://rapidsms.org/>`_).
```
Comment (adammichaelwood) [inline faq.rst]: These would be better presented in a list.

File: faq.rst [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+
+.. _what-android-use:
+
+What Android phone/tablet/device should I use? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Android ecosystem changes too rapidly to recommend one device. In general, we recommend you get devices that you can source in-country and run the latest Android OS (even though we support Android OS 1.6+ and higher). 
+ODK Collect will run on most Android form factors (including tablets and netbooks).
+
+We recommend you spend a little more to get a higher quality device, instead of buying the cheapest phone. If you need a supplier, try amazon.com, newegg.com, ebay.com, n1wireless.com, expansys-usa.com and negrielectronics.com.
```
Comment (adammichaelwood) [inline faq.rst]: list and hyperlink

File: faq.rst [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+
+For a list of all Android devices, try wikipedia.org, gsmarena.com, and phonescoop.com.
```
Comment (adammichaelwood) [inline faq.rst]: list and hyperlink

File: faq.rst [hunk 13]
```diff
@@ -0,0 +1,179 @@
+FAQ
+==============
+
+Truth be told, these questions aren't frequently asked. Hopefully the answers are still helpful.
```
Comment (adammichaelwood) [inline faq.rst]: Delete this.

File: faq.rst [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+Open Data Kit (ODK) is a free and open source set of tools which help organizations create mobile data collection solutions.
+
+Our `blog <https://opendatakit.org/blog/>`_ blog and `deployments page <https://opendatakit.org/about/deployments/>`_ have good examples of what others have used ODK for. Our `research page <https://opendatakit.org/about/research/>`_ also has videos and slides that explain ODK.
+
+.. _how-use: 
+
+How do I use ODK?
+~~~~~~~~~~~~~~~~~~~~
+
+Please read the :doc:`Getting Started Guide <getting-started>` section of our documentation to understand the initiou instructions.
```
Comment (adammichaelwood) [inline faq.rst]: initiou ?

Comment (ArthurSampaio) [reply faq.rst]: typo.

**Pull request merged**